tests/components/application-connector/test/compass-runtime-agent/synchronisation_test.go

```diff
@@ -19,33 +19,34 @@ type ApplicationReader interface {
 }
 
 func (gs *CompassRuntimeAgentSuite) TestCreatingApplications() {
-
 	// Created in chart
 	expectedAppName := "app1"
 	compassAppName := expectedAppName + random.RandomString(10)
 
-	// Create Application in Director and wait until it gets created
+	//Create Application in Director and wait until it gets created
 	applicationInterface := gs.applicationsClientSet.ApplicationconnectorV1alpha1().Applications()
-	runtimeID, err := gs.createAppAndWaitForSync(applicationInterface, compassAppName, expectedAppName)
+	applicationID, err := gs.createAppAndWaitForSync(applicationInterface, compassAppName, expectedAppName)
 	gs.Require().NoError(err)
 
 	// Compare Application created by Compass Runtime Agent with expected result
 	err = gs.appComparator.Compare(expectedAppName, compassAppName)
 	gs.Require().NoError(err)
 
 	// Clean up
-	err = gs.directorClient.UnregisterApplication(runtimeID)
+	err = gs.directorClient.UnregisterApplication(applicationID)
 	gs.Require().NoError(err)
 }
 
 func (gs *CompassRuntimeAgentSuite) createAppAndWaitForSync(appReader ApplicationReader, compassAppName, expectedAppName string) (string, error) {
 
-	var runtimeID string
+	var applicationID string
+
+	scenarioName := "auto-testing"
 
 	exec := func() error {
-		id, err := gs.directorClient.RegisterApplication(compassAppName)
+		id, err := gs.directorClient.RegisterApplication(compassAppName, scenarioName)
```

Comment:
Shouldn't it also be assigned to the correct formation to appear in the runtime? I didn't check it manually yet, just reading the code

Reply:
I'm not sure since I did not write this particular piece. We will see during tests

Reply:
Ok